### PR TITLE
Radha

### PR DESCRIPTION
**Pull request by RadhaArul**

### CODE REVIEW

File: Garage 2.0/Automapper/GarageMappings.cs [hunk 1]
```diff
@@ -7,7 +8,7 @@ public class GarageMappings : Profile
     {
         public GarageMappings()
         {
-            CreateMap<Member, MemberCreateViewModel>().ReverseMap();
+           // CreateMap<Member, MemberCreateViewModel>().ReverseMap();
```
Comment (cisci86) [inline Garage 2.0/Automapper/GarageMappings.cs]: Have you comment out the mapping? Why?

Comment (RadhaArul) [reply Garage 2.0/Automapper/GarageMappings.cs]: thats old one i uncomment that

Comment (RadhaArul) [reply Garage 2.0/Automapper/GarageMappings.cs]: i have solved that comment

File: Garage 2.0/Models/MemberOverViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿#nullable disable
+namespace Garage_2._0.Models
+{
+    public class MemberOverViewModel
+    {
+       public string SocialSecurityNumber { get; set; }
+       public string FirstName { get; set; }
+       public string Name { get; set; }
```
Comment (cisci86) [inline Garage 2.0/Models/MemberOverViewModel.cs]: Is this Last name? then it should say last name. Or full name?

**Review by IronPikachu — APPROVED** [review]

**Review by IronPikachu — APPROVED** [review]

**Review by IronPikachu — APPROVED** [review]

**Review by cisci86 — APPROVED** [review]

**Pull request merged**